### PR TITLE
Update privacy policy & h2/h3 tag readability

### PR DESCRIPTION
**Pull request by critzo**
This PR updates the Privacy Policy page and increases the size of h2 tags for better outline readability.
Addresses #350 and #377.
These changes can be previewed here: https://critzo.github.io/m-lab.github.io/privacy/

---
This change is [Reviewable](https://reviewable.io/reviews/m-lab/m-lab.github.io/380)

### CODE REVIEW

File: _pages/privacy.md [hunk 1]
```diff
@@ -7,11 +7,6 @@ title: "M-Lab Privacy Policy"
 # Measurement Lab Privacy Policy
 {:.no_toc}
 
-This Policy was last modified on _August 4, 2017_
```
Comment (georgiamoon) [inline _pages/privacy.md]: Shouldn't we keep an updated date as a best practice? or no?

**Comment by georgiamoon** [comment]
One comment re: Updated Date, but LGTM.

**Comment by critzo** [comment]
@georgiamoon I wondered about leaving that in. The draft document did not include it, so I removed. I agree though, we should keep it.

**Comment by critzo** [comment]
@georgiamoon updated. PTAL?

**Pull request merged**